### ISSUE TITLE
Documentation: Integration with pytest-xdist

### ISSUE THREAD

**Issue by fkromer** [HypothesisWorks/hypothesis#1283]
If I want to use `pytest-xdist` to [speed up test runs by sending tests to multiple cpus](https://github.com/pytest-dev/pytest-xdist#speed-up-test-runs-by-sending-tests-to-multiple-cpus) the tests are distributed by `pytest` test, not by `hypothesis` example I guess?

**Comment by DRMacIver** [comment]
Yeah the integration with `pytest-xdist` is pretty much "there isn't any". `pytest-xdist` will work as normal, but as far as `pytest` is concerned, a `@given` decorated test is just one test.

We might add support for running Hypothesis tests in multiple processes at some point, but it probably won't be via `pytest-xdist`

**Comment by DRMacIver** [comment]
> Alright 🙂. Would have been nice to parallelize some really slow tests to allow migration from "nightly" test jobs to jobs run more frequently.

You could run them more frequently with a small value for `max_examples` set and only run them with a large number nightly.

> I'll check out the release notes/changelog to not miss it.

FWIW I wouldn't hold your breath. This is very much in the long tail of "stuff that might happen soon if someone funds it but if not is probably not happening in the next couple of years". Too many things to do, too few dev-hours to do them in!

**Comment by fkromer** [comment]
> You could run them more frequently with a small value for max_examples set and only run them with a large number nightly.

Right. Thx.

> This is very much in the long tail of "stuff that might happen soon if someone funds it but if not is probably not happening in the next couple of years". Too many things to do, too few dev-hours to do them in!

Business as usual :smile:
